### PR TITLE
Add more fields option profile page

### PR DESCRIPTION
**Pull request by Shujaat89**

### CODE REVIEW

File: services/updatework.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Created by shujaatali on 28/03/16.
+ */
+exports.updateWorks = function(editedWork,res){
+  var db = require('nano-blue')('http://localhost:5984/members');
+    db.atomic("cbs", "updateWork", "68a91f7c9772c99d58e93c038b000131",
```
Comment (omeryousaf) [inline services/updatework.js]: @Shujaat89 hardcoded string tou change karo for user id.

File: services/updatework.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        {field: "work", value: editedWork}, function (error, response) {
+            if (error) console.log(error);
```
Comment (omeryousaf) [inline services/updatework.js]: error k case main bhi return karao.

**Comment by omeryousaf** [comment]
LGTM :+1:

**Comment by Shujaat89** [comment]
:)

**Pull request merged**